multicore_runtime/dispatcher.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""A WSGI app that, once configured, dispatches requests to user apps."""
+
+import httplib
+import logging
+import re
+
+from werkzeug.wrappers import Request
+from werkzeug.wrappers import Response
+
+ERROR_TEMPLATE = '<h1>{code} {message}</h1>\n'
+
+def response_for_error(code):
```

Comment:
Docstring please.

I think `status` is better than `code`, at least that's what the Response named argument is.

You could use `http_status` for instance if you want to make it more explicit what kind of status it is.

Otherwise looks great, this is clearly reusable for returning HTTP errors.